src/DynamoCoreWpf/packages.config

```diff
@@ -6,7 +6,7 @@
   <package id="HelixToolkit.Wpf" version="2015.1.629" targetFramework="net45" />
   <package id="HelixToolkit.Wpf.SharpDX" version="2015.1.629" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net45" />
+  <package id="RestSharp" version="105.2.3" targetFramework="net45" requireReinstallation="true" />
```

Comment:
Question out of curiosity; this change says `targetFramework="net45"`, but there are other changes going from "v4.5" to "v4.7", is there a corresponding targetFramework "net47" value, or is it known that "net45" is correct?

Reply:
this package and version does not have a net 47 build - but it does a net 46 one. If I reinstall we can ask for target 47 and we'll get 46. I think we should probably update this to a newer version which includes a .net 47 target to take advantage of TLS updates as this in the dll that actually makes REST calls for us.

I think it should out of scope for this task though and be part of the TLS task we've filed in the backlog:
https://jira.autodesk.com/browse/QNTM-5552

Reply:
ended up looking into this here:
https://github.com/DynamoDS/PackageManagerClient/pull/22

I will do the same here in this PR.